index.js

```diff
@@ -81,7 +81,8 @@ testRunner([
 			return validator;
 		},
 		test: function (instance, json, schema) {
-			return instance.validate(json, schema).valid;
+			// third argument is `breakOnError` to halt validation at first error
+			return instance.validate(json, schema, true).valid;
```

Comment:
The reason for this is also that themis is using a whole host of special settings in their benchmarks, and testing all kinds of configurations with different stuff disabled. I don't want to venture down that slippery slope.